doc/source/whatsnew/v1.0.0.rst

```diff
@@ -224,7 +224,7 @@ Other enhancements
 - :meth:`DataFrame.sort_values` and :meth:`Series.sort_values` have gained ``ignore_index`` keyword to be able to reset index after sorting (:issue:`30114`)
 - :meth:`DataFrame.sort_index` and :meth:`Series.sort_index` have gained ``ignore_index`` keyword to reset index (:issue:`30114`)
 - :meth:`DataFrame.drop_duplicates` has gained ``ignore_index`` keyword to reset index (:issue:`30114`)
-- Added new writer for exporting Stata dta files in version 118, ``StataWriter118``.  This format supports exporting strings containing Unicode characters (:issue:`23573`)
+- Added new writer for exporting Stata dta files in version 118 and 119, ``StataWriterUTF8``.  This format supports exporting strings containing Unicode characters (:issue:`23573`)
```

Comment:
Already released 1.0 so will want to move to the 1.1 whatsnew when that comes available

Reply:
Missed that. This is unfortunate. Are deprecation cycles still OK since this renames a class to avoid creating a trivial class for the 119 version?

Reply:
Hmm so you want to rename what was added in 1.0 to this right? Maybe OK as an exception if you would just immediately deprecate anyway but cc @TomAugspurger for thoughts

Reply:
That was the idea -- it is a very trivial change to enable the new format, but StataWriter118 doesn't make sense when it supports format 119 too. 

I've gone down the deprecation path to 2.0 deprecation.  TBH I don't think (m)any will use StataWriter118 directly since `to_stata` contains virtually all of the functionality. 

I can revert this if quick deprecation/renaming is better.

Reply:
If it's simpler I think OK to just change since this was just released in the RC and not do a deprecation cycle, but @jreback as well for thoughts

Reply:
Ok, I can get a restored version back pretty quickly. I think avoiding deprecation would be good if it isn't too controversial.

Reply:
To confirm, `StataWriter118` was new in 1.0? If so, then I think we're OK to change the name for 1.0.

Reply:
@TomAugspurger yes, it is new in 1.0.0.  I have moved back to changing the class name for 1.0.0 with no reference for 1.1.0, and so no deprecation.